clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/ops/servergroup/AbstractEnableDisableKubernetesAtomicOperation.groovy

```diff
@@ -99,15 +105,24 @@ abstract class AbstractEnableDisableKubernetesAtomicOperation implements AtomicO
         throw new KubernetesOperationException("Server group failed to reach a consistent state. This is likely a bug with Kubernetes itself.")
       }
     }
-    
+    def application
+    def clusterName
     if (replicationController) {
-      pods = credentials.apiAdaptor.getReplicationControllerPods(namespace, description.serverGroupName)
+      application = replicationController.metadata.labels.get("app")
```

Comment:
I understand that part, but I am referring https://github.com/spinnaker/clouddriver/blob/master/clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/ops/servergroup/AbstractEnableDisableKubernetesAtomicOperation.groovy#L60-L61

If we can come up something better in this area.